composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -308,6 +308,7 @@ def handle_composer_errors(error)
             # If there *is* a lockfile we can't confidently distinguish between
             # cases where we can't install and cases where we can't update. For
             # now, we therefore just ignore the dependency.
+            Dependabot.logger.error(error.message)
```

Comment:
I noticed that elsehwere when logging `error.message` we generally also log the backtrace: https://github.com/search?q=repo%3Adependabot%2Fdependabot-core%20Dependabot.logger.error&type=code

I'm not sure what is in `error.message`... does the user have enough information in this situation to skip the backtrace?

Just want to be sure that the user has enough information to understand what triggered the error since it can be due to two very different reasons:
1. lockfile can't be checked because install fails due to a missing native extension, ie "can't install"
2. lockfile has conflicting version requirements, ie "can't update"